<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1372</width>
    <height>846</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>1351</width>
      <height>511</height>
     </rect>
    </property>
    <property name="title">
     <string>Video live feed</string>
    </property>
    <widget class="QLabel" name="image_label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>1301</width>
       <height>431</height>
      </rect>
     </property>
     <property name="text">
      <string>Video</string>
     </property>
    </widget>
    <widget class="QPushButton" name="StartVideo">
     <property name="geometry">
      <rect>
       <x>610</x>
       <y>480</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Start</string>
     </property>
    </widget>
    <widget class="QPushButton" name="StopVideo">
     <property name="geometry">
      <rect>
       <x>750</x>
       <y>480</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Stop</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1372</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
